stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/attachment/AttachmentSelectionDialogFragment.kt

```diff
@@ -158,16 +159,16 @@ public class AttachmentSelectionDialogFragment : BottomSheetDialogFragment(), At
         internal const val REQUEST_KEY_FILE_MANAGER = "key_file_manager"
         internal const val BUNDLE_KEY = "bundle_attachments"
 
-        internal var staticStyle: AttachmentSelectionDialogStyle? = null
+        internal var staticStyle: MessageInputViewStyle? = null
 
         /**
          * Create a new instance of the Attachment picker dialog.
          *
          * See [AttachmentSelectionDialogStyle.createDefault] to load a default set of icons to be used for the
          * attachment dialog's tabs.
          */
-        public fun newInstance(attachmentSelectionDialogStyle: AttachmentSelectionDialogStyle): AttachmentSelectionDialogFragment {
-            staticStyle = attachmentSelectionDialogStyle
+        public fun newInstance(style: MessageInputViewStyle): AttachmentSelectionDialogFragment {
```

Comment:
This class is public but we need a style object in order to create it. `AttachmentSelectionDialogStyle` has `createDefault` function which make it easy to create, but what about `MessageInputViewStyle`? Do we have an easy way to create/pass it from somewhere? 
Moreover, theoretically, this is a breaking change but the dialog fragment is marked with `@ExperimentalStreamChatApi` annotation so it should be fine. Let's just mention this change in CHANGELOG.

Reply:
Good catch. I've added `MessageInputViewStyle::createDefault` and updated changelog.